onboard/nav/stateMachine.hpp

```diff
@@ -121,6 +116,9 @@ class StateMachine
     // Search pointer to control search states
     Searcher* mSearcher;
 
+    // Avoidance pointer to control obstacle avoidance states
+    Avoidance* mObstacle;
```

Comment:
This comment and variable name don't seem to explain what this does. Maybe `mObstacleAvoidanceStateMachine` or something similar